main.js

```diff
@@ -2,5 +2,127 @@
  * Created by Jenya on 18.04.15.
  */
 
+/// At first I'm creating general BeingClass. It will be the top of all being classes.
+function BeingClass(clas) {
+    this.clas = clas;
```

Comment:
I changed the logic of it. Now every class of lower level (e.g. Animal, HumanBeing etc.) has a 'clas' property that holds its class name.
I made this change because I decided that 'name' property should be set at lower level constructors. 
While writing these lines, I realized that every instance of BeingClass would have the 'name' property. Thus this decision increased the amount of lines and violated the DRY rule.

So I'll make the 'name' property to be set at BeingClass constructor.